### PR TITLE
Vet Clinic : DB from Schema design

### PR DESCRIPTION
**Pull request by jeanbuhendwa**
In this pull request, we have : 

- [ ] Created a file named schema_based_on_diagram.sql where we implemented the database from the diagram.
- [ ] Joined tables from many-to-many relationships
- [ ] Added the FK indexes.
- [ ] Created join tables and their indexes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -99,6 +99,13 @@ You can deploy this project using: **Not available**.
 - Twitter: [@jeanbuhendwa](https://twitter.com/jeanjacqueI)
 - LinkedIn: [LinkedIn](https://www.linkedin.com/in/johnbuhendwa/)
 
+👤 **Damilare Adepoju**
+
+- GitHub: [@githubhandle](https://github.com/adamilare)
+- Twitter: [@twitterhandle](https://twitter.com/mailtodare)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/adamilare)
```
Comment (Munsa1) [inline README.md]: - [ ] Kindly fix the broken link to your `LinkedIn` account, as broken links are not allowed
image

File: schema_based_on_diagram.sql [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+
+-- Create medical history table 
+DROP TABLE IF EXISTS medical_histories;
+CREATE TABLE medical_histories(
+	id INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+	admitted_at TIMESTAMP,
+	patient_id INT REFERENCES patients(id),
+	status varchar(100)
+);
```
Comment (Munsa1) [inline schema_based_on_diagram.sql:25]: - [ ] It's a requirement that all "join tables" for the many-to-many relationships are created, even if they are not shown on the diagram. Therefore, you should create a table between medical_histories and treatments

File: schema_based_on_diagram.sql [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+	treatment_id INT REFERENCES treatments(id)
+);
+
+CREATE INDEX patient_id ON medical_histories(patient_id)
+
+CREATE INDEX medical_history_id ON invoices(medical_history_id)
+
+CREATE INDEX treatment_id ON invoice_items(treatment_id)
+
+CREATE INDEX invoice_id ON invoice_items(invoice_id)
```
Comment (Munsa1) [inline schema_based_on_diagram.sql:54]: - [ ] Note that after you create the join table between the `medical_histories` and `treatments` table, remember to also create `INDEXES` for the two foreign keys in that join table

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @jeanbuhendwa @adamilare, 
required-changes-3

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
Created All Visible Tables With Primary Keys :heavy_check_mark: 
Descriptive PR :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by 1GORDON — APPROVED** [review]
# Status: APPROVED 🟢 

Hi @jeanbuhendwa & @adamilare ✋

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! :tada: :clap: :clap: :100:

## Highlights!
- Followed Gitflow ✔️
- Good commit messages ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend considering them as they can improve your code._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**